test.cpp

```diff
@@ -1764,6 +1782,8 @@ int main(int argc, char **argv)
     TEST_CASE(simplifyPath_cppcheck);
     TEST_CASE(simplifyPath_New);
 
+    TEST_CASE(convertCygwinPath);
```

Comment:
please put it in alphabetical order. It seems we put the `simplifyPath` in the wrong order before.